SourceX/discord/discord.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef DISCORD
+
+#include <string>
+
+#define DISCORD_DIABLO_APPID 496571953147150354

```

Comment:
Unused

Reply:
I'd rather keep it because it is Discord's id for Diablo, in case it can be used later.